src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,11 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_ACCEPTABLE, reason = "Book and id mismatch")
```

Comment:
It is good practice to have all error messages as constants in a separate class, we should not have them literal in the place where the exception is raised

Reply:
I gotchu my man